libbeat/processors/add_kubernetes_metadata/config.go

```diff
@@ -30,12 +30,11 @@ type Enabled struct {
 
 type PluginConfig []map[string]common.Config
 
-func defaultKuberentesAnnotatorConfig() kubeAnnotatorConfig {
+func defaultKubernetesAnnotatorConfig() kubeAnnotatorConfig {
 	return kubeAnnotatorConfig{
 		InCluster:       true,
 		SyncPeriod:      1 * time.Second,
 		CleanupTimeout:  60 * time.Second,
-		Namespace:       "kube-system",
```

Comment:
I think it's best if we remove `Namespace` field from the kubeAnnotatorConfig, and move all references to `client.Namespace`

Reply:
Yeah, I considered doing that initially as well. Will adjust now!